sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosRowConverter.scala

```diff
@@ -257,6 +269,8 @@ private object CosmosRowConverter
               parseOperationType(objectNode)
             case StructField(CosmosTableSchemaInferrer.TtlExpiredAttributeName, BooleanType, _, _) =>
               parseTtlExpired(objectNode)
+            case StructField(CosmosTableSchemaInferrer.LsnAttributeName, LongType, _, _) =>
```

Comment:
Please add tests in the existing Spec for this.

Reply:
Done in next iteration